package.json

```diff
@@ -1,55 +1,31 @@
 {
-  "name": "@readme/eslint-config",
-  "version": "8.5.1",
-  "description": "ReadMe coding standards",
-  "main": "index.js",
-  "author": "Jon Ursenbach <jon@ursenba.ch>",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:readmeio/eslint-config.git"
-  },
-  "engines": {
-    "node": "^12 || ^14 || ^16"
-  },
+  "name": "standards",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Core coding and style standards for all ReadMe technology.",
   "scripts": {
+    "lint": "npx lerna run lint --stream",
     "prepare": "husky install",
-    "prettier": "prettier --list-different --write \"./**/**.js\"",
-    "release": "npx conventional-changelog-cli -i CHANGELOG.md -s && git add CHANGELOG.md",
-    "test": "eslint ."
+    "publish": "npx lerna publish",
+    "test": "npx lerna run test --stream",
+    "version": "npx conventional-changelog-cli --pkg lerna.json -i CHANGELOG.md -s && git add CHANGELOG.md"
   },
-  "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.14.0",
-    "@typescript-eslint/parser": "^5.14.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-plugin-chai-expect": "^3.0.0",
-    "eslint-plugin-chai-friendly": "^0.7.2",
-    "eslint-plugin-eslint-comments": "^3.1.2",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jest": "^26.0.0",
-    "eslint-plugin-jest-dom": "^4.0.0",
-    "eslint-plugin-jest-formatting": "^3.0.0",
-    "eslint-plugin-jsdoc": "^38.0.3",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-mocha": "^10.0.3",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-testing-library": "^5.0.0",
-    "eslint-plugin-unicorn": "^41.0.0",
-    "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0"
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/readmeio/standards.git"
   },
-  "peerDependencies": {
-    "eslint": "^8.0.0",
-    "prettier": "^2.0.2"
+  "bugs": {
+    "url": "https://github.com/readmeio/standards/issues"
   },
+  "homepage": "https://github.com/readmeio/standards#readme",
+  "workspaces": [
+    "./packages/*"
+  ],
   "devDependencies": {
-    "@commitlint/cli": "^16.1.0",
-    "@commitlint/config-conventional": "^16.0.0",
-    "husky": "^7.0.1"
+    "@commitlint/cli": "^16.2.3",
+    "@commitlint/config-conventional": "^16.2.1",
```

Comment:
In addition to this config, I highly recommend we also [@commitlint/config-lerna-scopes](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-lerna-scopes), which requires the commit "scope" to match the sub-package name. This then lets Lerna during publish to associate commits and add them to each sub-package's respective changelog.

Reply:
I'm definitely up for requiring scoped commits but we should run this by the larger eng org first.

Reply:
To clarify, this won't require commit messages to be scoped. But if you choose to provide scope, then the scope has to match one of its sub-packages.

e.g.
```
// This passes
fix: something broken

// This is an error
fix(balloon): something broken

// This passes
fix(eslint-config): tweak eslint rules
```

Reply:
Ah sick, yeah we should do that but we might still want to run it by folks first.

Reply:
Sounds good to me. What's the best way to communicate this and get folx's opinions on it? #dev channel? Or eng meeting?

Reply:
Yeah probably a poll in #dev would be good enough.

Reply:
Just removed it for now